app-drawer-layout/app-drawer-layout.html

```diff
@@ -233,16 +233,22 @@
             contentContainer.style.marginRight = '';
           }
         }
-        if (drawer) {
+        if (drawer && this._drawerNeedsReset) {
```

Comment:
There seems to be an issue if you only get one resize event when switching to/from narrow (e.g. by toggling dev tools):

![image](https://cloud.githubusercontent.com/assets/1319720/18798188/29b77da2-8186-11e6-86d0-4b6d279daa5f.png)

Then after a single resize event:

![image](https://cloud.githubusercontent.com/assets/1319720/18798233/56dae68e-8186-11e6-9df9-39a1b08aea04.png)

I guess it's a timing issue - `marginLeft` needs to be set before header does `getBoundingClientRect`.